packages/template-gallery/src/__tests__/gallery-template-assets.test.ts

```
@@ -192,7 +202,8 @@ describe('gallery template assets', () => {
 	it('maps every bundled gallery template to a committed preview image', async () => {
 		const { getGalleryPreviewSrc } = await import('../preview-assets');
 
-		for (const key of getBundledGalleryKeys()) {
+		expect(DISPLAY_KEYS.size, 'display templates are bundled, so the exemption is live').toBeGreaterThan(0);
```

ESLint

[error] Replace `DISPLAY_KEYS.size,·'display·templates·are·bundled,·so·the·exemption·is·live'` with `⏎↹↹↹DISPLAY_KEYS.size,⏎↹↹↹'display·templates·are·bundled,·so·the·exemption·is·live'⏎↹↹`

(prettier/prettier)